indexer/bin/indexer.js

```diff
@@ -88,12 +88,27 @@ export default {
         pdpVerifierAddress: PDP_VERIFIER_ADDRESS,
       })
 
+      /**
+       * @param {BigInt} setId
+       * @param {string} rootId
+       * @returns
+       */
+      const maybeGetRootCid = async (setId, rootId) => {
+        try {
+          const cid = await pdpVerifier.getRootCid(setId, BigInt(rootId))
+          return cid
+        } catch (/** @type {any} */ err) {
+          console.error(
+            `Cannot get root CID for setId=${setId} rootId=${rootId}: ${err?.stack ?? err}`,
+          )
+          return null
```

Comment:
I think we don't want it to fail silently, we wouldn't notice
```suggestion
          throw err
```